dockerfiles/Dockerfile.arm64v8-dockerx

```diff
@@ -0,0 +1,79 @@
+# Get arm64 dependency
```

Comment:
There is already a dockerfiles/Dockerfile.arm64v8 file, In my opinion, if we are going to get rid off qemu binary emulation, we should then use that file and adapt it instead.

Reply:
@sarandia another change I'd like to address on this proposal is to add dockerfile/Dockerfile.armv8 dockerfile/Dockerfile.armv8-master and dockerfile/Dockerfile.armv8-debug in the same line of amd64.

Reply:
The DFLB flags are the same in Dockerfile.arm64v8-buildx and Dockerfile.arm64v8, so they should produce the same image.

Can we submit a follow-up PR to merge these two Dockerfiles and update the build-release.yaml? The Dockerfile.arm64v8-buildx should be built using `docker buildx build --platform linux/arm64`. Therefore, the build-release.yaml file needs some revamp, and I am currently uncertain about the syntax and how we can test it.

Essentially, the current `docker build` command in build-release.yaml needs to be replaced by `docker buildx build`, and it needs a new flag --platform with the correct values (arm64, amd64, arm32 as opposed to the current arch names arm64v8, arm32v7, x86_64).

Reply:
@sarandia OK, lets stick this PR to the proposed Cmake changes, which overall seems correct. Please remove the proposed addition of dockerfiles/Dockerfile.arm64v8-dockerx and lets start a new PR to address the switch to buildx independently.

Reply:
OK, done, removed this Dockerfile, and will add you to a new PR for buildx changes.

Reply:
Great. Please update the commit message and pull request description accordingly.

Reply:
Updated.